packages/camera/camera_android_camerax/android/src/main/java/io/flutter/plugins/camerax/CameraXProxy.java

```diff
@@ -5,7 +5,11 @@
 package io.flutter.plugins.camerax;
 
 import android.app.Activity;
+import android.graphics.SurfaceTexture;
+import android.view.Surface;
 import androidx.camera.core.CameraSelector;
+import androidx.camera.core.Preview;
+import io.flutter.plugin.common.BinaryMessenger;
 
 public class CameraXProxy {
```

Comment:
Non blocking. 
The best time to add documentation is when you are in the code modifying the class. Consider adding some non obvious context in javadocs as you go.